library/src/main/java/com/devbrackets/android/playlistcore/service/PlaylistServiceCore.java

```diff
@@ -738,6 +738,20 @@ protected void performPlay() {
                 videoPlayer.play();
             }
         }
+        
+        //Immediately pauses the media
```

Comment:
I din't got what you mean with:
> clean up the methods called from startMediaPlayer() to correctly abide by the seek and immediately pause (which should be renamed to startPaused with the re-work).

Are you saying to rewrite the `startMediaPlayer` method to not call performPlay(), and instead call a another method that handles correctly seek and immediately pause?